mlflow/pyfunc/__init__.py

```diff
@@ -313,6 +313,7 @@
 DATA = "data"
 ENV = "env"
 MODEL_CONFIG = "config"
+DATABRICKS_FEATURE_LOOKUP = "databricks-feature-lookup"
```

Comment:
This will show up as a public attribute in the `mlflow.pyfunc` package. I'd prefer that we move this inside the `_warn_dependency_requirement_mismatches` function, since:

A. It's only used there
B. This variable doesn't have a `_` prefix so will show up in code completion when a user looks for attributes of the `mlflow.pyfunc` module